--- OpenGL-0.5/Makefile.PL.orig	Sat Aug  3 17:36:50 2002
+++ OpenGL-0.5/Makefile.PL	Sat Aug  3 18:11:00 2002
@@ -15,8 +15,7 @@
 
 $INC = "-I/usr/X11R6/include";
 			      
-@LIBS = qw(-L/usr/X11R6/lib -L/usr/local/lib -lGL -lGLU -lGLUT -lGLX 
--lMesaGL -lMesaGLU -lMesaGLUT -lMesaGLX -lglut -lXext -lXmu -lXi -lICE -lXIE -lX11);
+@LIBS = qw(-L/usr/X11R6/lib -lGL -lGLU -lglut -lX11);
 
 WriteMakefile(
     'NAME'	=> 'OpenGL',
