--- OpenGL-0.7006/Makefile.PL.orig	2025-04-14 06:33:29.000000000 +0200
+++ OpenGL-0.7006/Makefile.PL	2025-09-20 21:38:13.179157843 +0200
@@ -78,34 +78,13 @@ our $NO_LGCC = grep /^NO_LGCC$/, @ARGV;
 
 my $prepare_success = eval {
 # Look for available libs
-our @libdirs = map "-L$_", grep -d, qw(
-  /usr/X11R6/lib
-  /opt/X11/lib
-  /usr/local/lib
-  /usr/openwin/lib
-  /opt/csw/lib
-  /opt/local/lib
-  /usr/local/freeglut/lib
-  /usr/lib
-  /usr/lib/xorg/modules
-  /usr/X11R6/lib/modules
-  /usr/lib/xorg/modules/extensions
-  /usr/X11R6/lib/modules/extensions
-);
+our @libdirs;
 
 our @check_libs = qw
 {
   -lGL
-  -lopengl
-  -lMesaGL
   -lGLU
-  -lglu
-  -lMesaGLU
-  -lfreeglut
-  -lglut3
   -lglut
-  -lGLUT
-  -lMesaGLUT
 };
 
 our $found_libs = get_found_libs(\@libdirs,@check_libs);
@@ -181,8 +181,7 @@ WriteMakefile(
       "lib/OpenGL/Config.pm ".
       "gl_exclude.h ".
       "utils/glversion.txt ".
-      "utils/glversion$Config{exe_ext} ".
-      "utils/glversion$Config{obj_ext}"
+      ""
   },
   %$build_config,
   INC => $INCS . ' -I'.curdir(),
@@ -504,6 +503,7 @@ sub get_gldata {
     ($found->{FREEGLUT32} || $found->{FREEGLUT}) ? "GLUT_DEF=HAVE_FREEGLUT " :
     ($found->{GLUT32} || $found->{GLUT}) ? "GLUT_DEF=HAVE_GLUT " :
     '';
+=cut
   # Platform-specific makefiles for glversion
   my $make_ver;
   my $append = " LIB='$libs' INC='$incs' DEF='$defs'";
@@ -541,6 +541,10 @@ sub get_gldata {
   }
   die "get_gldata: no extensions found in $glv_file\n" if !keys %$gldata;
   print "This looks like OpenGL Version: $gldata->{VERSION}\n";
+=cut
+  use Config;
+  my $gldata = { };
+
   if ($^O eq 'darwin') {
     $gldata->{VERSION} = '4.1';
     print "  for MacOS X, overriding to version $gldata->{VERSION}\n";
@@ -572,11 +576,13 @@ sub generate_excl {
   }
   # Create gl_exclude.h
   die "Unable to write to $exc_file\n" if !open my $excl_fh, ">", $exc_file;
+=cut
   print $excl_fh "// OpenGL Extension Exclusions - may be modified before building.\n";
   print $excl_fh "//\n";
   print $excl_fh "// Generated for ".$gldata->{VENDOR}.", ".$gldata->{RENDERER}."\n";
   print $excl_fh "// OpenGL v".$gldata->{VERSION}.", using ";
   print $excl_fh 'FreeGLUT v'.$gldata->{FREEGLUT}."\n" if $gldata->{FREEGLUT};
+=cut
   my $GL_VERSION = $gldata->{VERSION} =~ m|^(\d\.\d+)| ? $1 : undef;
   # Make an empty exclusion file if a Windows distribution build
   print $excl_fh "\n";
@@ -618,7 +597,7 @@ sub generate_excl {
 
 sub get_defines {
   my ($found_libs, $interface_lib, $incs) = @_;
-  my $defs = '-DHAVE_VER';
+  my $defs = '';
   $defs .= " -DIS_STRAWBERRY" if $IS_STRAWBERRY;
   print "resulting libs:\n" if $verbose;
   foreach my $key (sort keys %$found_libs) {
@@ -642,7 +642,6 @@ sub get_defines {
     $out = cfile_text('GL/freeglut.h', $INCS);
     $defs .= " -DHAVE_FREEGLUT -DHAVE_FREEGLUT_H"
       if $out =~ m|glutMainLoopEvent|;
-    $defs .= " -DGL_GLEXT_LEGACY";
   }
   $defs;
 }
