diff -durN OpenGL-0.5.orig/Makefile.PL OpenGL-0.5/Makefile.PL
--- OpenGL-0.5.orig/Makefile.PL	Sun Apr 25 02:16:47 1999
+++ OpenGL-0.5/Makefile.PL	Fri May 24 08:59:33 2002
@@ -13,6 +13,8 @@
 $DEFS .= " -DHAVE_GLX";       # Interface to X/Windows for OpenGL --
                               # emulated on several non-X systems.
 
+$INC = "-I/usr/X11R6/include";
+			      
 @LIBS = qw(-L/usr/X11R6/lib -L/usr/local/lib -lGL -lGLU -lGLUT -lGLX 
 -lMesaGL -lMesaGLU -lMesaGLUT -lMesaGLX -lglut -lXext -lXmu -lXi -lICE -lXIE -lX11);
 
@@ -23,6 +25,7 @@
 	'XSPROTOARG' => '-noprototypes',
 	'OBJECT' => '$(BASEEXT)$(OBJ_EXT) gl_util$(OBJ_EXT)',
 	'LIBS' => join(' ', @LIBS),
+	'INC' => $INC,
 	'PM' => {'OpenGL.pm' => '$(INST_LIBDIR)/OpenGL.pm'},
 	'dist' => { COMPRESS=>"gzip", SUFFIX=>"gz" }
 );
